### ISSUE TITLE
How do I make my updated hexchat have some looks from the previous version?

### ISSUE THREAD

**Issue by brunoais** [hexchat/hexchat#533]
This new version of hexchat has things I miss from previous version (2.9.5).
Basically I wanted it to have the same look and feel I had with the previous hexchat (2.9.2). Personally, I don't like this new theme and I didn't expect I'd lose everything I had changed with the update.

I wanted to remove that annoying vertical bar that appears in the chat windows that devides the username and the text.
I wanted to recover the logs I had in my %appdata%\hexchat\logs and make them appear again when I open a channel tab.
I wanted to separate the notices into different tabs.

Basically I wanted all (or almost all) to have the looks of hexchat 2.9.2
How can I do that? Is there a theme with that? If there is, which one is it (I already know what there is to know to install themes)?

EDIT:
I already kinda removed that that vertical bar. using text_show_sep but what it did was to hide the bar, not remove it altogether like the 2.9.2 version...
Help is requested I don't like these kinds of changes I was forced to make :(
